test_data/test_route/texts/README.md

```diff
@@ -33,7 +33,7 @@ Step to update standard mapping:
 {
```

Comment:
the existence of this file is a bit irritating to me ;-)

Reply:
Why?

Reply:
a readme in a test files subfolder indicates a weird/over-engineered setup

Reply:
I can simply delete, but this probably doesn't solve the problem, does it? If not, out of the scope ;)